app/models/user.rb

```diff
@@ -232,10 +232,6 @@ def groups_include?(group)
     miq_group_ids.include?(group.id)
   end
 
-  def admin?
-    userid == "admin"
```

Comment:
Are there cases in the UI where the code literally does `== "admin"`.  If so, perhaps it's better to go through this accessor than leave those things in the UI.  The reason I suggest that is that there are known requests around us having the "admin" not be an EVM user, but instead a user that can be pulled in via LDAP.